src/adapter/deno/deno.d.ts

```diff
@@ -17,7 +17,10 @@ declare namespace Deno {
    */
   export function writeFile(path: string, data: Uint8Array): Promise<void>
 
-  export function upgradeWebSocket(req: Request): {
+  export function upgradeWebSocket(
+    req: Request,
+    options: any
```

Comment:
Add ` // eslint-disable-next-line @typescript-eslint/no-explicit-any` above it to avoid eslint error.